python/ray/setup-dev.py

```diff
@@ -121,15 +121,6 @@ def do_link(package, force=False, skip_list=None, local_path=None):
     do_link("widgets", force=args.yes, skip_list=args.skip)
     do_link("cluster_utils.py", force=args.yes, skip_list=args.skip)
     do_link("_private", force=args.yes, skip_list=args.skip)
-    # Link package's `dashboard` directly to local (repo's) dashboard.
-    # The repo's `dashboard` is a file, soft-linking to which will not work
-    # on Mac.
-    do_link(
-        "dashboard",
-        force=args.yes,
-        skip_list=args.skip,
-        local_path="../../../dashboard",
```

Comment:
you should just remove line 131 and keep the other lines.

it is basically listing all the directories under ray to `setup-dev`.

Reply:
fixed.

verified it is linked correctly too:

```
Creating symbolic link from
.../ray-project/ray/python/ray/dashboard to
.../anaconda3/envs/py3/lib/python3.10/site-packages/ray/dashboard
```